.github/workflows/ci.yml

```diff
@@ -26,6 +26,11 @@ jobs:
       with:
         go-version: 1.15
 
+    - name: Setup Node
```

Comment:
I think you need to do the same for the `build-*.yml` workflows as well

besides that LGTM

Reply:
Done.